src/EventStore.Core/Util/Opts.cs

```diff
@@ -363,6 +363,9 @@ public static class Opts {
 
 		public const string UnsafeAllowSurplusNodesDescr = "Allow more nodes than the cluster size to join the cluster as clones. (UNSAFE: can cause data loss if a clone is promoted as master)";
 		public static readonly bool UnsafeAllowSurplusNodesDefault = false;
+		
+		public const string DeadMemberRemovalTimeoutDescr = "Causes an isolated node to be drop by the cluster";
```

Comment:
This description may be a bit confusing as the node would be removed from the gossip rather than the cluster.

Perhaps something more like:

```
How long in {seconds/minutes} a dead node will remain in the gossip before being pruned
```